mu-plugins/central-hub/tests/phpunit/unit/meta-data/autoloadConfigurations.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Tests for the function autoload_configurations().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\autoload_configurations;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_AutoloadConfigurations
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_AutoloadConfigurations extends Test_Case {
+
+	/**
+	 * Array of default config params.
+	 *
+	 * @var array
+	 */
+	private $defaults = [];
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/module.php';
+
+		$this->defaults = (array) require CENTRAL_HUB_ROOT_DIR . '/src/meta-data/defaults/meta-box-config.php';
+		$this->defaults = current( $this->defaults );
+	}
+
+	/**
+	 * Test autoload configurations() should load a metabox config from the filesystem into Config Store.
+	 */
+	public function test_should_load_config_from_filesystem_into_config_store() {
+		$config_file = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/meta-box-members-runtime-config.php';
+
+		// Assert that loadConfigFromFilesystem() runs once and receives the expected parameters.
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\loadConfigFromFilesystem' )
```

Comment:
@rgadon107 Notice that this `expect` assertion is the test for this method. We're testing that the dependent function `loadConfigFromFilesystem()` runs only once and it receives the parameters we expect it to receive.

Does this make sense to you?